sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Linq/DigitalTwinsQueryBuilder.cs

```diff
@@ -0,0 +1,406 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Globalization;
+using System.Text;
+using Azure.Core;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder.Linq
+{
+    /// <summary>
+    /// Azure DigitalTwins Query builder that facilitates writing queries against ADT instances. Redirects to generic version of this class using BasicDigitalTwin as type T.
+    /// </summary>
+    public class DigitalTwinsQueryBuilder : DigitalTwinsQueryBuilder<BasicDigitalTwin>
+    {
+        /// <summary>
+        /// Create a DigitalTwins query and set the queried collection to DigitalTwins by default.
+        /// </summary>
+        public DigitalTwinsQueryBuilder() : base(DigitalTwinsCollection.DigitalTwins) { }
```

Comment:
Nit - consider using ``/// <inherit />``` as the entire doc comment for these and then you won't need to keep the doc comments in sync:
```suggestion
        /// <inherit />
        public DigitalTwinsQueryBuilder() : base(DigitalTwinsCollection.DigitalTwins) { }
```